src/Sylius/Component/Core/Model/UserInterface.php

```diff
@@ -36,4 +36,36 @@ public function removeAuthorizationRole(RoleInterface $role);
      * @return bool
      */
     public function hasAuthorizationRole(RoleInterface $role);
+
+    /**
+     * @return string
+     */
+    public function getEmail();
+    
+    /**
+     * @param string $email
+     */
+    public function setEmail($email);
+    
+    /**
+     * Gets normalized email (should be used in search and sort queries).
+     *
+     * @return string
+     */
+    public function getEmailCanonical();
+    
+    /**
+     * @param string $emailCanonical
+     */
+    public function setEmailCanonical($emailCanonical);
```

Comment:
The email field and methods should be in UserBundle. In Core we just substitute `return $this->email` with `return $this->customer->getEmail()`.